package.json

```diff
@@ -63,6 +62,7 @@
     "tslib": "^2.3.1",
     "uuid": "^8.3.2",
     "webdav": "^4.8.0",
+    "xliff": "^6.0.3",
```

Comment:
move this to `devDependencies` once `json` translation file is created at compile time.